library/build.go

```diff
@@ -120,6 +120,7 @@ func (b *Build) Environment() map[string]string {
 		// add the deployment target to the list
 		envs["VELA_BUILD_TARGET"] = target
 		envs["VELA_DEPLOYMENT"] = target
+		envs["BUILD_TARGET"] = target
```

Comment:
Match the existing environment variable injected via the compiler https://github.com/go-vela/compiler/blob/master/compiler/native/environment.go#L235.